atsign-foundation/atsign.dev-3.0
# gitignore hugo_stats.json (generated file)

**- What I did**

Ignoring `hugo_stats.json` which gets regenerated with each hugo build:

- This should help prevent merge conflicts which @CurtlyCritchlow experienced today.
- The file is used as a purgeCSS safelist

**- How I did it**

Local Changes

**- How to verify it**

See that the staging site still builds correctly on merge, this file should be generated by hugo at build time in the workflow.

**- Description for the changelog**
gitignore hugo_stats.json (generated file)
